### ISSUE TITLE
Price on Chart 10% lower

### ISSUE THREAD

**Issue by jaime10a** [Gbox4/tstock#15]
The candles on the chart seem to be about 10% lower on every stock.
Probably the prices are right, but the y-axis labeling is somehow off.
I have everything in USD, but it could also be that it somehow gets converted to EURO.(Cause the exchange is about 10% lower)

**Comment by Gbox4** [comment]
I believe that the issue you're describing is the same as #8, is that correct? Even when you pull data from an exchange that doesn't use USD, it still prints a "$" dollar sign. I'll fix this soon

**Comment by jaime10a** [comment]
I don't think so. I was using the default settings and checking stocks from US market.
I just downloaded from aur and did 'tstock nvda' for example. And all stocks the graph is wrong.
Maybe it's cause I'm requesting from European IP?

**Comment by Gbox4** [comment]
Hmmmm still can't seem to replicate this behavior. Can you attach a screenshot of the output?
